datafiles/xed-isa.txt

```diff
@@ -8861,7 +8861,7 @@ ATTRIBUTES : SKIPLOW64  REQUIRES_ALIGNMENT
 PATTERN   : 0x0F 0x15 no_refining_prefix MOD[mm] MOD!=3 REG[rrr] RM[nnn]  MODRM()
 OPERANDS  : REG0=XMM_R():rw:ps MEM0:r:dq
 PATTERN   : 0x0F 0x15 no_refining_prefix MOD[0b11] MOD=3 REG[rrr] RM[nnn]
-OPERANDS  : REG0=XMM_R():rw:ps REG1=XMM_B():r:dq
+OPERANDS  : REG0=XMM_R():rw:ps REG1=XMM_B():r:q
```

Comment:
Gonna reject  this one. Another developer saw  this and also raised concerns.  The way XED  is expressing  this is as if 128b of the  reg1 operand  were read, but the SKIPLOW64  is there to  tell you  only the upper 64b  are referenced.  I  see what you  are  trying to  say -- q  is more  accurate than  dq  if you are thinking about  how much data  are referenced. The  intent  of  SKIPLOW64  was to communicate the same thing as what you  are trying  to get at.

Reply:
Sure, no problem but then we will need to change operand sizes for other instructions with SKIPLOW64. They have 'q' at the moment while only this one has 'dq'. BTW, what's about MOVH there you have 'q' as well while high part is getting read. Should we introduce 'hq' or something similar then?